spec/models/audit/event/authn/authenticate_spec.rb

```diff
@@ -56,6 +65,16 @@
       expect(subject.severity).to eq(Syslog::LOG_WARNING)
     end
 
+    it 'contains the not-found user field' do
```

Comment:
shouldn't we have another use-case?
`success = false` & role is found in the DB => role_id is in the audit message.

note that in order to do that you will need to mock the `Role` class.